spec/support/matchers/match_valkyrie_ids_with_af_ids.rb

```diff
@@ -1,5 +1,19 @@
-RSpec::Matchers.define :match_valkyrie_ids_with_active_fedora_ids do |_active_fedora_ids|
-  match do |valkyrie_ids|
-    contain_exactly(valkyrie_ids.map(&:id))
+module RSpec
+  module Matchers
+    module BuiltIn
```

Comment:
this is a minor nitpick, but shouldn't this module be reserved for the built-in RSpec matchers? any reason not to put `MatchValkyrieIdsWithActiveFedoraIds` in the top-level namespace?

Reply:
I tried putting it in a different namespace and it wasn’t picked up by RSpec.  I’m not sure how it brings in matchers.

Reply:
hmmm... i assumed that https://github.com/samvera/hyrax/pull/3902/files#diff-c7c616878ab53fcfcec45311b43d0da2R13-R17 took care of registering this with RSpec? maybe i'm missing something?

do you mind if i give it a try?

Reply:
Found a fix for the namespace issue.  Thanks for the push back.  This is better.